core/src/test/java/org/kiwix/kiwixmobile/core/history/viewmodel/HistoryStateTest.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.core.history.viewmodel
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.kiwix.kiwixmobile.core.history.adapter.HistoryListItem.DateItem
+import org.kiwix.kiwixmobile.core.history.viewmodel.effects.historyItem
+import org.kiwix.kiwixmobile.core.history.viewmodel.effects.historyState
+
+internal class HistoryStateTest {
+  @Test
+  internal fun `isInSelectionMode is true when item is selected`() {
+    assertThat(
```

Comment:
seems the formatting spread many things over a few more lines than necessary?

Reply:
ah, that's annoying.